server/src/main/scala/Routes.scala

```diff
@@ -0,0 +1,18 @@
+package server
+
+import akka.http.scaladsl.server.Directives
+import akka.http.scaladsl.server.Route
+import java.io.File
+
+object Routes extends Directives {
+  def apply(): Route =
+    pathSingleSlash(getFrom("index.html")) ~
+    path(Segment)(getFrom) ~
+    path("target" / Segment)(s => getFrom(s"target/$s"))
+
+  def getFrom(content: String): Route =
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594775828/issues/source?bid=3301982&fileBranchId=3306761#l13)